requestOption.js

```diff
@@ -43,6 +43,13 @@ SuiteRequestOption.prototype = {
       .concat([headerToSet]);
   },
 
+  getHeader: function(name) {
+    var result = this.headers
+      .filter(this._exactHeader(name));
```

Comment:
You can use find instead of filter.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find#Examples